src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -65,6 +65,13 @@ function isDisabled(node) {
 	return node.disabled;
 }
 
+function isChecked(node) {
+	if (node.checked) {
+		return true;
+	}
+	return node.false;
```

Comment:
[Well](https://stackoverflow.com/questions/17320961/how-do-i-simplify-an-if-statement-that-returns-true-or-false) .... why not just 
```suggestion
	return node.checked || false;
```
if you are worried about `undefined` see [here](https://stackoverflow.com/questions/21367903/javascript-false-undefined-vs-undefined-false)